app/components/utils.js

```diff
@@ -41,19 +41,19 @@ exports.forceHttps = function(req, res, next) {
 
 exports.getStore = function (redisConfig, session) {
     if (redisConfig.enabled === 'true') {
-        const ioRedis = require('ioredis');
+        const Redis = require('ioredis');
         const RedisStore = require('connect-redis')(session);
         let client;
-        if (redisConfig.useTLS) {
-            client = ioRedis.createClient(redisConfig.port,
-                {'password': redisConfig.password,
-                    'host': redisConfig.host,
+        if (redisConfig.useTLS === 'true') {
```

Comment:
Perhaps we can use a ternary for the options rather than instantiating Redis in each part of the if statement, something like:

```
const tlsOptions = {
    password: redisConfig.password,
    port: redisConfig.port,
    tls: true
};
const options = redisConfig.useTLS === 'true' ? tlsOptions : {};
const client = new Redis(redisConfig.port, redisConfig.host, options);
```